pdf_to_ppt/setup.py

```diff
@@ -0,0 +1,13 @@
+setup(
+  name="pdf_to_ppt",
+  version="0.1.0",
+  description="Converts PDF to PowerPoint",
+  url="https://github.com/luan-dev/tools",
+  author="Luan <github.com/luan-dev>",
+  license="GPLv3",
+  packages=["converter"],
+  include_package_data=True,
+  install_requires=[
+        "PyQt5",
```

Comment:
add all of you install requirements here so if you do the pip install -e . setup.py will have all of your dependents

Reply:
this was an artifact of my pyqt5 attempt